crates/bevy_pbr/src/prepass/prepass.wgsl

```diff
@@ -142,24 +142,6 @@ fn vertex(vertex_no_morph: Vertex) -> VertexOutput {
 }
 
 #ifdef PREPASS_FRAGMENT
-struct FragmentInput {
-#ifdef VERTEX_UVS
-    @location(0) uv: vec2<f32>,
-#endif // VERTEX_UVS
-
-#ifdef NORMAL_PREPASS
-    @location(1) world_normal: vec3<f32>,
-#endif // NORMAL_PREPASS
-
-#ifdef MOTION_VECTOR_PREPASS
-    @location(3) world_position: vec4<f32>,
-    @location(4) previous_world_position: vec4<f32>,
-#endif // MOTION_VECTOR_PREPASS
-
-#ifdef DEPTH_CLAMP_ORTHO
-    @location(5) clip_position_unclamped: vec4<f32>,
-#endif // DEPTH_CLAMP_ORTHO
-}
```

Comment:
It isn’t exactly the same. The meaning of the built-in position is different between vertex output and fragment input and the values will be different. That could be fine, and just mean that the member name should be renamed from clip_position to position or something. Then I see that the tangent is also omitted from the fragment input. Perhaps just because it wasn’t used?

Reply:
To clarify a bit, we found a bug that was caused because the `FragmentInput` didn't have the vertex tangents defined when the normal prepass is used with a custom shader. After adding them we noticed that it's identical to VertexOutput. The only difference being that `@builtin(position) cliip_position` was never defined in FragmentInput.

Reply:
We do this a lot with MeshVertexOutput everywhere else in bevy so I think it makes sense to do this here too

Reply:
but it is true that `clip_position` isn't the correct name for the fragment shader, we can change it to `position` if that would help. I see that in `MeshVertexOutput` it is called `position`.
what do you think  @IceSentry?

Reply:
Yeah, I'm fine with that part matching `MeshVertexOutput` it makes sense. I'd also suggest to name it `PrepassMeshVertexOutput`. This way it makes it very clear that it's the prepass version.

Reply:
will do

Reply:
done